Makefile

```diff
@@ -475,7 +475,7 @@ CLIENT_GEN ?= $(LOCALBIN)/client-gen
 GIT_CLONE    := git clone
 
 # External generation tool versions
-CONTROLLER_TOOLS_VERSION ?= v0.14.0
+CONTROLLER_TOOLS_VERSION ?= v0.16.5
```

Comment:
you don't need this, since it was already bumped here: https://github.com/containers/nri-plugins/commit/b51c35bb921d83986d3daf0b3e9891d8ccb5304b